clients/fides-js/src/components/ConsentBanner.tsx

```diff
@@ -26,19 +27,25 @@ const ConsentBanner: FunctionComponent<BannerProps> = ({
   waitBeforeShow,
 }) => {
   const [isShown, setIsShown] = useState(false);
+  const hasMounted = useHasMounted();
```

Comment:
ah right I wanted to move this `hasMounted` logic to `App.tsx` instead and then we should only need to do it once in `App.tsx`. I'll do that in https://github.com/ethyca/fides/pull/3291